crates/sparrow-runtime/src/read/parquet_stream.rs

```diff
@@ -70,7 +70,7 @@ pub(super) async fn new_parquet_stream(
         let batch = RecordBatch::try_new(projected_schema.clone(), raw_batch.columns().to_vec())
             .into_report()
             .change_context(Error::ReadingBatch)
-            .attach_printable_lazy(|| RecordBatchAttachment::new("raw_batch", &raw_batch))
+            // .attach_printable_lazy(|| RecordBatchAttachment::new("raw_batch", &raw_batch))
```

Comment:
the `json` write did not support printing out `maps`, and was failing. But, I suppose that's only an issue if the `try_new` fails anyway. Will uncomment

Reply:
Also next version should fix it: https://github.com/apache/arrow-rs/pull/4364/files